_shared/project/conf/supervisord.conf

```diff
@@ -3,27 +3,37 @@ nodaemon=true
 environment=PYTHONUNBUFFERED="1"
 logfile=/dev/null
 logfile_maxbytes=0
-{% if cookiecutter.get("_directory") == "pyapp" %}
+{% if include_exists("conf/supervisord.conf.json") %}
```

Comment:
This block of "duplicated" template code has the same structure and logic in `supervisord-dev.conf` and `supervisord.conf` but differs in some ways:

* The `supervisord.conf` template reads `supervisord.conf.json` whereas the `supervisord-dev.conf` template reads `supervisord-dev.conf.json`
* The set of default settings and values injected into each `[program:foo]` section differs between dev and prod
* The default lists of programs used if the project has no `supervisord.conf.json` or `supervisord-dev.conf.json` file are different

Unfortunately with cookiecutter there's no way to share template code or macros between two different template files in the `cookiecutters` repo. I can't say 100% for sure that there isn't some unintended way that we can hack around this but I don't think we should, especially not just for this one case.